themes/src/main/resources-community/theme/base/account/messages/messages_es.properties

```diff
@@ -144,4 +201,158 @@ invalidPasswordMinUpperCaseCharsMessage=Contrase\u00F1a incorrecta: debe contene
+applicationInUse=In-use app only
+clearAllFilter=Lipiar todos los filtros
+activeFilters=Filtros activos
+filterByName=Filtar por nombre
+allApps=Todas las aplicaciones
+internalApps=Aplicaciones internas
+thirdpartyApps=Aplicaciones de terceros
+appResults=Resultados
+clientNotFoundMessage=Cliente no encontrado.
+
+# Linked account
+authorizedProvider=Proveedor autorizado
+authorizedProviderMessage=Proveedores autorizados vinculados a su cuentat
+identityProvider=Proveedores de identidad
+identityProviderMessage=Para vincular su cuenta con proveedores de identidad que haya configurado
+socialLogin=Iicio de sesi\u00F3n con redes sociales
+userDefined=Definido por el usuario
+removeAccess=Eliminar acceso
+removeAccessMessage=Deber\u00E1 volver a otorgar acceso si desea utilizar esta cuenta de aplicaci\u00F3n.
+
+#Authenticator
+authenticatorStatusMessage=La autenticaci\u00F3n de dos factores esta activa actualmente
+authenticatorFinishSetUpTitle=Su autenticaci\u00F3n de dos factores
+authenticatorFinishSetUpMessage=Cada vez que inicie sesi\u00F3n en su cuenta de Keycloak, se le pedir\u00E1 que proporcione un c\u00F3digo de autenticaci\u00F3n de dos factores.
+authenticatorSubTitle=Configurar la autenticaci\u00F3n de dos factores
+authenticatorSubMessage=Para mejorar la seguridad de su cuenta, active al menos uno de los m\u00E9todos de autenticaci\u00F3n de dos factores disponibles.
+authenticatorMobileTitle=Autenticador m\u00F3vil
+authenticatorMobileMessage=Use el Autenticador m\u00F3vil para obtener c\u00F3digos de verificaci\u00F3n como autenticaci\u00F3n de dos factores.
+authenticatorMobileFinishSetUpMessage=El autenticador se ha vinculado a su tel\u00E9fono.
+authenticatorActionSetup=Configurar 
+authenticatorSMSTitle=C\u00F3digo SMS
+authenticatorSMSMessage=Keycloak enviar\u00E1 el c\u00F3digo de verificaci\u00F3n a su tel\u00E9fono como la autenticaci\u00F3n de dos factores.
+authenticatorSMSFinishSetUpMessage=Los mensajes de texto se env\u00EDan a
+authenticatorDefaultStatus=Defecto
+authenticatorChangePhone=Cambiar n\u00FAmero de tel\u00E9fono
+authenticatorBackupCodesTitle=C\u00F3digos de respaldo
+authenticatorBackupCodesMessage=Obtenga sus c\u00F3digos de respaldo de 8 d\u00EDgitos
+authenticatorBackupCodesFinishSetUpMessage=Se generaron 12 c\u00F3digos de respaldo en este momento. Cada uno se puede usar una vez.
+
+#Authenticator - Mobile Authenticator setup
+authenticatorMobileSetupTitle=Configuraci\u00F3n del autenticador m\u00F3vil
+smscodeIntroMessage=Ingrese su n\u00FAmero de tel\u00E9fono y se le enviar\u00E1 un c\u00F3digo de verificaci\u00F3n.
+mobileSetupStep1=Instale una aplicaci\u00F3n de autenticaci\u00F3n en su tel\u00E9fono. Las aplicaciones enumeradas aqu\u00ED son compatibles.
+mobileSetupStep2=Abra la aplicaci\u00F3n y escanee el c\u00F3digo QR:
+mobileSetupStep3=Ingrese el c\u00F3digo de un solo uso proporcionado por la aplicaci\u00F3n y haga clic en Guardar para finalizar la configuraci\u00F3n.
```

Comment:
"Introduzca" again.

Reply:
In my country, "Ingrese" and "Introduzca" are used interchangeably, but perhaps this is not the case in the other areas, it may be better to change it to "Introduzca"